### PR TITLE
Logs of login and logout

### PR DESCRIPTION
**Pull request by durbar**
Description:
Describe your pull request and attach links to any issues that may be related.

### CODE REVIEW

File: server/utils/commonFunctions.js [hunk 1]
```diff
@@ -67,6 +68,26 @@ async function verifyJWT(token) {
...
+
+	const details = await verifyJWT(token);
+
+	if(!details.error)
+		return details;
+
+	if(details.error && details.message != 'jwt expired')
+		return details;
+
+	let token_details = [];
```
Comment (durbar) [inline server/utils/commonFunctions.js]: sadf

File: server/www/session-logs.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	async insert() {
+
+		const db = dbConfig.write.database.concat('_logs');
+
+		const userAgent = this.request.body.user_agent || this.request.get('user-agent');
+
+		const user_agent = new commonFun.UserAgent(userAgent);
+
+		let params = {
+			user_id: this.request.body.user_id,
```
Comment (durbar) [inline server/www/session-logs.js]: zfh

File: server/www/session-logs.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+			const token_details = await commonFun.getUserDetailsJWT(params.refresh_token);
+
+			if(token_details.error)
+				return;
+
+			params.user_id = token_details.user_id;
+			params.session_id = token_details.sessionId;
+		}
+
+		delete(params['refresh_token']);
```
Comment (durbar) [inline server/www/session-logs.js]: .

File: client/js/main.js [hunk 4]
```diff
@@ -754,23 +755,42 @@ class User {
...
 
-		const parameters = new URLSearchParams();
+		try {
 
-		await Storage.clear();
+			const
+				parameters = {
+					user_id: user.user_id,
+					type: 'logout',
+					description: 'Logout from UI.',
```
Comment (achshar) [inline client/js/main.js]: Not needed.

**Pull request merged**